docs/source/manual/file_formats/bitstream_setting.rst

```diff
@@ -112,6 +113,29 @@ The following syntax are applicable to the XML definition tagged by ``non_fabric
 
   The final ``pb_type`` name is "bram.bram_lr[mem_36K_tdp].mem_36K"
 
-.. option:: ``pb`` child element content="<string>"
+.. option:: pb child element content="<string>"
 
   The content of the ``pb_type`` data to be extracted. For example, ``content=".param INIT_i"`` means that the data will be extracted from the ``.param INIT_i`` line defined under the ``.blif model``.
+
+bit-related Settings
+^^^^^^^^^^^^^^^^^^^^
+
+This is to allow user to set particular bit using full path in the hierarchy of FPGA fabric
```

Comment:
Mention that the overwrite has the highest priority than loading any external bitstream file

Reply:
Update doc via https://github.com/chungshien/OpenFPGA/commit/cbe9a46f95094dd3f0b6842685f9a83c8d4e08a6